package.json

```diff
@@ -25,7 +25,7 @@
     },
     "dependencies": {
         "@chili-publish/grafx-shared-components": "^0.56.0",
-        "@chili-publish/studio-sdk": "1.7.4",
+        "@chili-publish/studio-sdk": "1.7.10",
```

Comment:
there is more recent version then this one. Should we switch to it?

Reply:
sure!